### PR TITLE
Test for validateReview-function

### PR DESCRIPTION
**Pull request by oskber**
- I made a function to validate reviews before sending to CMS
- I made tests to verify that it works

### CODE REVIEW

**Review by maxlin94 — APPROVED** [review]
Looks good, nice work :)

**Review by ConnyPettersson — APPROVED** [review]
Looks Good!

File: utils/reviewsUtils.js [hunk 1]
```diff
@@ -72,3 +73,11 @@ export async function getAverageRating(adapter, id) {
   }
   return { rating: averageRating, maxRating: maxRating };
 }
+
+export async function validateReview(review) {
```
Comment (richardolsson) [inline utils/reviewsUtils.js:77]: You have created this function as an `async` function, but it does no asynchronous work. There can be reasons to do something like this, but I'm guessing it's a mistake.

File: routes/reviews.js [hunk 2]
```diff
@@ -19,11 +19,12 @@ const payload = await getAverageRating(cmsAdapter, req.params.id);
...
-    await postReview(cmsAdapter, req.body);
-    res.status(200).send({ message: 'Success' });
-  } catch (error) {
-    res.status(400).send({ message: error.message });
+  const validated = await validateReview(req.body);
+  if (validated === false) {
+    res.status(400).send({ message: 'Invalid input' });
+  } else {
+  await postReview(cmsAdapter, req.body);
+  res.send({ message: 'Success' });
```
Comment (richardolsson) [inline routes/reviews.js:27]: The code style here is a little bit sloppy, with indentation not matching the block that the code is in. I would expect the code on these two lines to be indented one more level.

**Review by richardolsson — COMMENTED** [review]
Some feedback below.

**Pull request merged**